english/Yu-Gi-Oh/yami yugi.md

```diff
@@ -0,0 +1,3 @@
+The young and steely-eyed Yami Yugi is an extraordinary and confident duelist who fights the ultimate battle at the Shadow Games.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.